orc8r/gateway/c/common/sentry/SentryWrapper.cpp

```diff
@@ -36,6 +36,7 @@
 #define SHARED_MCONFIG "shared_mconfig"
 #define SHOULD_UPLOAD_MME_LOG "sentry_upload_mme_log"
 #define MME_LOG_PATH "/var/log/mme.log"
+#define SYS_LOG_PATH "/var/log/magma_services.log"
```

Comment:
While reading this again: should we rename `SYS_LOG_PATH` to something that reveals that we are writing to this path and not reading from it? The current name might suggest that it is the path to the syslog file, analogous to  `MME_LOG_PATH`.